web/spiders/spider_ce.py

```diff
@@ -0,0 +1,65 @@
+import io
+from datetime import datetime
+
+import rows
+
+from .base import BaseCovid19Spider
+
+
+class Covid19CESpider(BaseCovid19Spider):
+    name = "CE"
+    start_urls = ["https://indicadores.integrasus.saude.ce.gov.br/api/coronavirus/qtd-por-municipio?tipo=Óbito,Confirmado"]
+
+    def parse(self, response):
+        encoding = "utf-8"
+        table = rows.import_from_json(
+            io.BytesIO(response.body),
+            encoding=encoding
+        )
+
+        # TODO - get last_date from another source at saude.ce.gov.br
+        last_date = datetime.now().date()
+        self.add_report(date=last_date, url=self.start_urls[0])
+
+        total_confirmed = total_deaths = 0
+        imported_confirmed = imported_deaths = None
+
+        cases = {}
+        for row in table:
+
+            if cases.get(row.municipio) == None :
+                cases[row.municipio] = {}
+
+            if row.tipo == 'Confirmado':
+                cases[row.municipio]["confirmed"] = row.quantidade
+                continue
+            
+            if row.tipo == 'Óbito':
+                cases[row.municipio]["deaths"] = row.quantidade 
+                continue
+        
+        for case in cases.items():
```

Comment:
Aqui você poderia usar o "tuple unpacking": `for city, city_data in cases.items():` - Python é bem legal, né? :)